validate.lic

```diff
@@ -355,6 +355,13 @@ class DRYamlValidator
     error("thanatology['ritual_type']: '#{settings.thanatology['ritual_type']}' is invalid, only [preserve, harvest, fetish, cut, dissect, consume, arise, and cycle] are supported at this time.")
   end
 
```

Comment:
Is it worth adding a validator for the case in which the user has `thanatology['store'] = false` but `necro_safe_room_use_material: true`?

Reply:
Done, good call.